shopinvader_customer_multi_user/README.rst

```diff
@@ -0,0 +1,88 @@
+===============================
+Shopinvader Customer Multi User
+===============================
+
+.. !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+   !! This file is generated by oca-gen-addon-readme !!
+   !! changes will be overwritten.                   !!
+   !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+
+.. |badge1| image:: https://img.shields.io/badge/maturity-Beta-yellow.png
+    :target: https://odoo-community.org/page/development-status
+    :alt: Beta
+.. |badge2| image:: https://img.shields.io/badge/licence-AGPL--3-blue.png
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+.. |badge3| image:: https://img.shields.io/badge/github-shopinvader%2Fodoo--shopinvader-lightgray.png?logo=github
+    :target: https://github.com/shopinvader/odoo-shopinvader/tree/12.0/shopinvader_customer_multi_user
+    :alt: shopinvader/odoo-shopinvader
+
+|badge1| |badge2| |badge3| 
+
+Allow registration of multiple user for the same company.
+
+**Table of contents**
+
+.. contents::
+   :local:
+
+Usage
+=====
+
+First of all you have to enable the feature:
+
+Shopinvader backend -> Sale tab -> Enable "Validate customers"
+
+Then the registration workflow changes as following:
+
+* for each company binding a unique token is generated, you can see it in the Shopinvader tab on company form
+* on Locomotive registration page users can enter the company token (if your theme does not support this yet, just add a text field named `company_token`)
```

Comment:
This addon doesn't depend on locomotive. we should explain how the addon works reagrding the service layer...